components/script/dom/htmlanchorelement.rs

```diff
@@ -600,16 +599,20 @@ pub fn follow_hyperlink(subject: &Element, hyperlink_suffix: Option<String>, ref
     };
 
     // Step 7.
-    let chosen = match target_attribute_value {
+    let (maybe_chosen, replace) = match target_attribute_value {
```

Comment:
Just changed this, so 'replace' is actually set to true if a new bc was created...